Programmimg_Problems 7

```diff
@@ -0,0 +1,11 @@
+for (var i = 1; i <= 10; i++) {
+  xTable(i);
+  console.log("");
+}
+
+function xTable(n) {
```

Comment:
Did you write this youself?

Reply:
I made reeesrch on "how to print a multiplication table using javascript" and used the idea i got from there